PyDSS/dssCircuit.py

```diff
@@ -2,7 +2,8 @@
 
 class dssCircuit(dssObjectBase):
 
-    VARIABLE_OUTPUTS_BY_LABEL = {}
+    VARIABLE_OUTPUTS_BY_LABEL = {
+        "AllBusMagPu":{"is_complex": True, "units": ['[pu]']}}
```

Comment:
@nadiavp I think that this is incorrect. This function returns an array of floats. I think line 7 is already correct. I plan to delete this. Let me know if you disagree.